### PR TITLE
Enhancement/fonts buttons

### PR DESCRIPTION
**Pull request by willthebrit**
- Reduces button corner radius to 2px
- Resizes default button, small, mini to 36px, 31px, 26px respectively 
- Adds system font stack
- Sets the body color to #333333
- Sets the body line-height to a unit-less 1.5
- Adds fontWeights to theme
- Darkens Secondary button to grey80
- Brightens up 'Yellow' color
- Removes GradientButton in favor of 'Flat' button
- Fixes $ alignment on Input
- Adds color to table header, align text left
- BulletList color

### CODE REVIEW

File: packages/hui-pages/src/routes/TextDocs/index.js [hunk 1]
```diff
@@ -57,17 +56,17 @@ const TextDocs = () => (
         <Text jumbo>jumbo</Text>
       </Col>
       <Col xs={3}>
-        <Text thinnest>thinnest</Text>
+        <Text larger thinnest>thinnest</Text>
```
Comment (billyxs) [inline packages/hui-pages/src/routes/TextDocs/index.js]: Did thinnest and thinner need to be larger than the other weighted examples?

File: packages/hui-pages/src/routes/TableDocs/index.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ const CheckboxDocs = () => (
       <Col xs={12}>
         <Code>
           {`import Table from '@hixme-ui/table'
-          
-<Table striped alignHeader='left' alignBody='right'>
+
+<Table striped blue alignHeader='left' alignBody='right'>
```
Comment (billyxs) [inline packages/hui-pages/src/routes/TableDocs/index.js]: Tables are looking snazzy

File: packages/button/src/FlatButton.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const FlatButton = ButtonBase.extend`
+  background: ${props => getButtonFlatColor(props)};
+  border: 2px solid rgba(255, 255, 255, 0);
+  border-radius: ${props => getButtonBorderRadius(props)};
+  * {
+    cursor: pointer;
+  };
+  :hover:enabled {
+    filter: brightness(110%);
```
Comment (billyxs) [inline packages/button/src/FlatButton.js]: wha? 😮

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "lerna": "^2.9.0"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist",
+    "bootstrap": "npm run clean && lerna bootstrap --hoist --loglevel verbose",
```
Comment (billyxs) [inline package.json]: 👍

File: packages/hui-pages/src/routes/TextDocs/index.js (hunk 1 above)
Comment (willthebrit) [reply packages/hui-pages/src/routes/TextDocs/index.js]: Yep - you can't see the thinner and thinnest at 'normal', needs to be 'larger'

**Pull request merged**